msal4j-sdk/src/integrationtest/java/com.microsoft.aad.msal4j/IntegrationTestHelper.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import java.net.MalformedURLException;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class IntegrationTestHelper {
+
+    static PublicClientApplication createPublicApp(String appID, String authority) {
+        try {
+            return PublicClientApplication.builder(
+                            appID).
+                    authority(authority).
+                    build();
+        } catch (MalformedURLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    static void assertTokenResultNotNull(IAuthenticationResult result, boolean checkAccessToken, boolean checkIDToken) {
+        assertNotNull(result);
+        if (checkAccessToken) assertNotNull(result.accessToken());
```

Comment:
I am not aware of any case where the access token is null?

Reply:
Me neither, I just added it because there were a few tests where we just checked if the result or the ID token was null but not the access token.

However, I just used a 'true' value for both token options in all the tests that now use this, so in the latest commit I changed this helper method to be `assertAccessAndIdTokensNotNull` and it will always check both tokens.